.github/workflows/test.yml

```
@@ -0,0 +1,32 @@
+on: [push]
+name: Check buildability
```

```suggestion
name: Check build
```
I don't think that's a word.